.github/workflows/android_deploy_production_to_playstore.yml

```
@@ -0,0 +1,52 @@
+name: Android - Deploy Production build to Play Store
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_deploy_android:
+    name: Build & Deploy Android
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+
+      - name: Set up Java JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+
+      - name: Set up Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
+
+      - name: Get Flutter dependencies
+        run: flutter pub get
+
+      - name: Run code generator
+        run: flutter packages pub run build_runner build --delete-conflicting-outputs
+
+      - name: Set up .env
+        env:
+          ENV_PRODUCTION: $
+        run: |
+          echo $ENV_PRODUCTION > .env
+
+      - name: Build Production App Bundle
+        run: flutter build appbundle --flavor production --release --build-number $GITHUB_RUN_NUMBER
+
+      - name: Upload Android Production Release bundle to Play Store
+        uses: r0adkll/upload-google-play@v1
+        with:
+          serviceAccountJson: $
+          packageName: co.nimblehq.flutter.template
```

Should it be `ic.flutter.avishek`?
```suggestion
          packageName: co.nimblehq.ic.flutter.avishek
```

---

@Wadeewee since we'll not deploy to Google Play, I've decided to delete the file in this commit https://github.com/nimblehq/ic-flutter-avishek/pull/26/commits/4a5a28d7c48bca14d468e803f7e6f2875e5712d5